app-center-webapps/src/main/webapp/vueLocales/locale_fr.json

```diff
@@ -37,9 +37,8 @@
   "appCenter.adminSetupList.actions": "Actions",
   "appCenter.adminSetupList.remove": "Supprimer",
   "appCenter.adminSetupList.search": "Rechercher",
-  "appCenter.userSetup.appDirectory": "Annuaire des applications",
-  "appCenter.userSetup.authorized.access": "Accéder",
-  "appCenter.userSetup.authorized.favorite": "Favoris",
+  "appCenter.userSetup.appCenter": "Annuaire des applications",
+  "appCenter.userSetup.authorized.open": "OUVRIR",
```

Comment:
Why this label is in capital letter ?
Perhaps we should change it to normal, and capitalize it by css

Reply:
Okay will fix it to uppercase with css.